Formula/slicot.rb

```diff
@@ -0,0 +1,22 @@
+class Slicot < Formula
+  desc "Fortran 77 algorithms for computations in systems and control theory"
+  homepage "http://www.slicot.org"
+  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
+  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
+  version "5.0+20101122"
+  sha256 "fa80f7c75dab6bfaca93c3b374c774fd87876f34fba969af9133eeaea5f39a3d"
+
+  depends_on "gcc"
+
+  def install
+    args = [
+      "FORTRAN=#{ENV.fc}",
+      "LOADER=#{ENV.fc}",
+    ]
+    system "make", "lib", "OPTS=-fPIC", "SLICOTLIB=../libslicot_pic.a", *args
```

Comment:
Do we need `-fPIC` or is it a leftover from homebrew-science? (where it made sense on linux, but isn't necessary for us)

Reply:
Come to think of it, I'm pretty sure it was for Linux. Let's remove it.